### PR TITLE
Look for certificates in valet linux config directory

### PR DESCRIPTION
**Pull request by jameshulse**
Hi, this pull request adds another search path to support the [valet-linux](https://github.com/cpriego/valet-linux) project.

I understand that this project isn't officially supported, but a small change in this Vite plugin would allow our team to greatly simplify our Vite configuration and support developers using Linux or Mac with Herd.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -218,7 +218,7 @@ function resolveLaravelPlugin(pluginConfig: Required<PluginConfig>): LaravelPlug
                                 server.config.logger.info(`  ${colors.green('➜')}  Using Herd certificate to secure Vite.`)
                             }
 
-                            if (resolvedConfig.server.https.key.startsWith(valetConfigPath())) {
+                            if (resolvedConfig.server.https.key.startsWith(valetMacConfigPath()) || resolvedConfig.server.https.key.startsWith(valetLinuxConfigPath())) {
                                 server.config.logger.info(`  ${colors.green('➜')}  Using Valet certificate to secure Vite.`)
                             }
```
Comment (timacdonald) [inline src/index.ts:223]: @jameshulse, I've just made this change to ensure the Linux version also sees the log message in the Vite server output. Could you confirm that is working as expected?

**Review by timacdonald — APPROVED** [review]
These changes look good to me.

I've renamed the first-party valet function to `valetMac`. This is to make it easier to understand what is going on with both functions and also aligns with `herdMac` / `herdWindows`.

Although you could say that we want to keep the first-party method as just `valet`, it makes it easier to know there is another version to also check when working with the code.

File: src/index.ts (hunk 1 above)
Comment (jameshulse) [reply src/index.ts:223]: I sure do:

image

**Comment by jameshulse** [comment]
Good shout. Cheers for that Tim.

Shameless plug: if the team are looking for more early Laravel Cloud adopters, we are keen to get off self hosting by the end of the year.

File: src/index.ts (hunk 1 above)
Comment (timacdonald) [reply src/index.ts:223]: Thanks!

**Comment by taylorotwell** [comment]
Thanks @jameshulse!

**Pull request merged**